src/Backupper/BackupperFactory.php

```diff
@@ -58,6 +96,9 @@ public function create(?string $connectionName = null): AbstractBackupper
 
         \assert($backupper instanceof AbstractBackupper);
 
+        if (isset($this->excludedTables[$connectionName])) {
```

Comment:
Don't know why, but I prefer `\key_exists($connectionName, $this->excludedTables)`

Do you know if one is better than the other?